python-checks/src/test/resources/checks/regex/impossibleBackReferenceCheck.py

```diff
@@ -0,0 +1,9 @@
+import re
+
+pattern = re.compile(r"\1(.)", re.DOTALL) # Noncompliant {{Fix this backreference, so that it refers to a group that can be matched before it.}}
```

Comment:
It would be good to have some more tests (especially as we found that some finders behaviors could be different from one language to the other like with the `\w`) and to have at least one assertion on location.